validator/validators/performance/performance_validator.go

```diff
@@ -57,3 +90,153 @@ func (s *PerformanceValidator) Cleanup() error {
 
 	return nil
 }
+
+func (s *PerformanceValidator) SendPacketToDatabase(perfInfo PerformanceInformation) error {
+	var (
+		dataType               = s.vConfig.GetDataType()
+		receiver               = s.vConfig.GetPluginsConfig()
+		commitHash, commitDate = s.vConfig.GetCommitInformation()
+		agentCollectionPeriod  = fmt.Sprint(s.vConfig.GetAgentCollectionPeriod().Seconds())
+		// The secondary global index that is used for checking if there are item has already been exist in the table
+		// The performance validator will query based on the UseCaseHash to confirm if the current commit with the use case
+		// has been exist or not? If yes, merge it. If not, sending it to the database
+		// https://github.com/aws/amazon-cloudwatch-agent-test/blob/e07fe7adb1b1d75244d8984507d3f83a7237c3d3/terraform/setup/main.tf#L46-L53
+		kCheckingAttribute = []string{"CommitHash", "UseCase"}
+		vCheckingAttribute = []string{fmt.Sprint(commitHash), receiver}
+	)
+
+	err := backoff.Retry(func() error {
+		existingPerfInfo, err := awsservice.GetItemInDatabase(DynamoDBDataBase, "UseCaseHash", kCheckingAttribute, vCheckingAttribute, perfInfo)
+		if err != nil {
+			return err
+		}
+
+		// Get the latest performance information from the database and update by merging the existing one
+		// and finally replace the packet in the database
+		maps.Copy(existingPerfInfo["Results"].(map[string]interface{}), perfInfo["Results"].(map[string]interface{}))
```

Comment:
The reason why we want to merge is that there are many limitations on number of queries that DynamoDB can provide (e.g DynamoDB only provides maximum of [2 key queries in the database](https://stackoverflow.com/questions/65390063/dynamodbexception-conditions-can-be-of-length-1-or-2-only)). So, there are no ways for us to provide more granularity (e.g query commit hash and number of values per metric). Therefore, we need to merge all the performance metrics sending with 100 metrics/minute , 1000 metrics/minute, 5000 metrics/minute together. Afterwards, replace the existing database to avoid having duplicate key in the database.